app/src/main/java/io/appium/settings/Settings.java

```diff
@@ -70,12 +79,104 @@ public void onCreate(Bundle savedInstanceState) {
             LocationTracker.getInstance().start(this);
         }
 
+        // REQUIRED STEPS to activate screenrecord feature:
+        // adb shell pm grant io.appium.settings android.permission.RECORD_AUDIO
+        // adb shell pm grant io.appium.settings android.permission.WRITE_EXTERNAL_STORAGE
+        // adb shell appops set io.appium.settings PROJECT_MEDIA allow
+
+        // TO START SCREENRECORD:
+        // adb shell am start -n "io.appium.settings/io.appium.settings.Settings" -a io.appium.settings.recording.ACTION_START --es "recording_filename" "abc.mp4"
```

Comment:
Should this be start activity? I wonder if this feature can be via `broadcast`. (I haven't dug in the recording feature, so I might miss something)

Reply:
i tried the initiating recording with `broadcast` but since i had to trigger startActivityForResult() method supplied with mediaprojection intent object (see https://github.com/sirmordred/io.appium.settings/blob/master/app/src/main/java/io/appium/settings/Settings.java#L108) i had to use Settings activity

Reply:
but i will try to dig more on this